Create output/day15-fires.md with FEMA wildfire index documentation (#30)
zyang91/30-day-map-challenge-2025: output/day15-fires.md

# Day 15 of 30 Day Map Challenge - Fire

> **Day 15 (Fire)**: I created a visualization of wildfire risk across California counties, showcasing the spatial distribution of wildfire hazard levels throughout the state. This map uses a custom color palette transitioning from cool blues (low risk) through warm yellows to intense reds (high risk), effectively communicating the varying degrees of wildfire danger across California's diverse landscapes. The visualization highlights the elevated risk in inland and mountainous regions compared to coastal areas.

Reviewer: ```suggestion
**Day 15 (Fire)**: I created a visualization of wildfire risk across California counties, showcasing the spatial distribution of wildfire hazard levels throughout the state. This map uses a custom color palette transitioning from green (low risk) through warm yellows to intense reds (high risk), effectively showcasing the varying degrees of wildfire danger across California's diverse landscapes. The visualization highlights the elevated risk in inland and mountainous regions compared to coastal areas.
```

![](day15-fire.png)

**Technical Implementation:**
- **sf** - R package for handling spatial vector data and geometric operations
- **tidyverse** - R's ecosystem of packages for data manipulation and visualization
- **tigris** - R package for downloading TIGER/Line shapefiles from the US Census Bureau
- **ggplot2** - R's powerful data visualization package (via tidyverse)
- **grid** - R package for low-level graphics and layout control

**Data Sources:**
- **Wildfire Risk Data**: [FEMA National Risk Index (NRI)](https://hazards.fema.gov/nri/wildfire) - County-level wildfire risk scores (HWAV_RISKS field)
- **California Boundary**: US Census Bureau TIGER/Line Shapefiles via the tigris package

> **Coordinate Reference System:**

Reviewer: ```suggestion
```

> - `EPSG:3310` (California Albers Equal Area) for accurate representation of California geography

Reviewer: ```suggestion
```
